Dockerfile

```diff
@@ -11,3 +12,6 @@ RUN mkdir -p /tmp/buildpack/php /tmp/build_cache /tmp/env
 ADD https://github.com/heroku/heroku-buildpack-php/archive/$PHP_BUILDPACK_VERSION.tar.gz ./
 RUN tar -xzvf $PHP_BUILDPACK_VERSION.tar.gz -C /tmp/buildpack/php --strip-components 1 && rm $PHP_BUILDPACK_VERSION.tar.gz
 RUN /tmp/buildpack/php/bin/compile /app /tmp/build_cache /tmp/env
+
+# Set up xdebug
+RUN $HEROKU_PHP_BIN/pecl install channel://pecl.php.net/xdebug-2.4.0
```

Comment:
This actually builds the xdebug extension and puts it into the docker image.

Reply:
Using [pecl](https://pecl.php.net/) was mentioned at https://xdebug.org/docs/install. I found version 2.4.0 to be compatible with the version of PHP that we are using, i.e. PHP 5.